packages/config/config/devices/0x5254/bw8510cn.json

```diff
@@ -1,25 +0,0 @@
-// Remotec BW8510CN
-// ZRC-90CN
-{
-	"manufacturer": "Remotec",
-	"manufacturerId": "0x5254",
-	"label": "BW8510CN",
-	"description": "ZRC-90CN",
-	"devices": [
-		{
-			"productType": "0x000b",
-			"productId": "0x8510",
-			"zwaveAllianceId": 1585
```

Comment:
So this one doesn't exist?

Reply:
Correct. The zwa pulled it under the sku not the model number.